statistics/handle/handle_hist.go

```diff
@@ -104,6 +119,8 @@ func (h *Handle) SyncWaitStatsLoad(sc *stmtctx.StatementContext) bool {
 			}
 		case <-timer.C:
 			metrics.SyncLoadTimeoutCounter.Inc()
+			_, digest := sc.SQLDigest()
+			logutil.BgLogger().Warn("SyncWaitStatsLoad timeout", zap.String("digest", digest.String()))
```

Comment:
We can print the timeout value here.

Reply:
This is not necessary as we stored timeout duration in session vars already.